ee/packages/ddp-client/src/types/streams.ts

```diff
@@ -160,7 +160,6 @@ export interface StreamerEvents {
 			key: `${string}/webdav`;
 			args: [{ type: 'changed'; account: Partial<IWebdavAccount> } | { type: 'removed'; account: { _id: IWebdavAccount['_id'] } }];
 		},
-		{ key: `${string}/e2ekeyRequest`; args: [string, string] },
```

Comment:
We cannot simply remove this, this would be a breaking change, it's okay NOT to use this on the client side anymore but not to remove it from the server side, probably mobile is relying on this stream.